### ISSUE TITLE
Failed to resolve: com.github.AppIntro:AppIntro:6.1.0

### ISSUE THREAD

**Issue by Barengific** [AppIntro/AppIntro#991]
## 🐛 Describe the bug
Cannot sync Gradle properly. when I add the dependency and sync, I get the following error:

_Failed to resolve: com.github.AppIntro:AppIntro:6.1.0
Show in Project Structure dialog
Affected Modules: app_

## ✅ Expected behavior
It should sync without issues.

## 💣 Steps to reproduce
Created a new project, added the required dependencies then sync then problem shows.

## 📑 Your Code
App Module added:
```
dependencies {
    implementation 'androidx.core:core-ktx:1.3.2'
    implementation 'androidx.appcompat:appcompat:1.3.1'
    implementation 'com.google.android.material:material:1.4.0'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.1'
    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.3.1'
    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
    implementation 'androidx.navigation:navigation-fragment-ktx:2.3.5'
    implementation 'androidx.navigation:navigation-ui-ktx:2.3.5'
    testImplementation 'junit:junit:4.+'
    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
    implementation 'com.github.AppIntro:AppIntro:6.1.0'
}

```

Project build-> 
```
buildscript {
    repositories {
        google()
        mavenCentral()
        maven { url "https://jitpack.io" }
    }
    dependencies {
        classpath "com.android.tools.build:gradle:7.0.2"
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build. Gradle files
    }
}
```

## 📱 Tech info
 - AppIntro Version: 6.1.0 
 - Device: Doesn't sync to know which devices are affected
 - Graddle Version: 7.0.2
 - Kotlin plugin: 1.5.31
 - Compile SDK: 31

**Comment by paolorotolo** [comment]
Have a look at this, maybe you're in the same situation: https://github.com/AppIntro/AppIntro/issues/971

**Comment by Barengific** [comment]
Thanks for the response. I looked at the link and unfortunately, it didn't solve my issue.

However, I did solve it by adding:
`maven { url 'https://jitpack.io' }`
To my _settings.gradle_ instead of build.gradle:app or build.gradle:root
[Source.](https://stackoverflow.com/questions/68740655/i-didnt-see-allprojects-repositories) - Answer by Daniel Irungu.

Also sidenote, on the main page readme it says _AppIntroFragment.createInstance()_ to add slides but now it's updated to _AppIntroFragment.newInstance()_ for some reason this hasn't been updated.
